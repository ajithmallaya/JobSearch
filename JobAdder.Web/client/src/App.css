.App {
  text-align: center;
}

.App-header {
  background-image: linear-gradient(to left, #35c0ad, #35c0ad);
  text-align: center;
  text-decoration-style: solid;
  height: 40px;
  padding: 20px;
  color: white;
  font-size: 24px;
}

.App-title {
  font-size: 1.5em;
}

.App-intro {
  font-size: large;
  border: 1px solid #dedede;
  border-radius: 0;
  width:800px;
  margin-top: 40px;
  left:20%;
  position: absolute;
}

.jobs .jobs__item {
height: 465px;
width: 618px;
align-items: center;
}

.input-section {
  display: inline;
  background-color: goldenrod;
}
.input-section > div > span  {
margin-right: 10px;
}

.flex {
  display: flex;
  flex-direction: row;
  padding: 0;
}

.col-3 {
  width: 50%;
}

.text {
 display: block;
 text-align: left;
 padding: 3px;
}

.jobs .candidates-section {
  box-shadow: 1px 1px 1px 1px black;
  background-color: wheat;
  padding: 20px;
}

.jobs .jobs-section {
  box-shadow: 1px 1px 1px 1px #d2d3d3;
  color: #231f20;
  padding: 20px;
}

.jobs .job-header {
  border-top: 1px solid #d2d3d3;
  padding: 20px;
  color: #706D6E;
}

 .label-header {
  color: #706D6E;
}

.jobs .job-name ,
.jobs .candidate-name {
  font-weight: bold;
  font-size: 16px;
  color: #231f20;
}


@media (max-width: 600px) {
  .App-intro {
    left:0px;
  }
  .jobs .flex {
    flex-direction: column;
  }

  .jobs .col-3 {
    width: 100%;
    padding: 8px 0;
  }
}