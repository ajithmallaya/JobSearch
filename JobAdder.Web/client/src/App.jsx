import React, { Component } from "react";
import "./App.css";
import Jobs from './Jobs';

class App extends Component {
  

  render() {
    return (
      
      <div className="App">
      <div className="App-header"> Job Adder
      </div>
        <div className="App-intro">
         <Jobs />
        </div>
      </div>
    );
  }
}

export default App;
