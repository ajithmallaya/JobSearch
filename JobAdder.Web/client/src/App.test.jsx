import { shallow } from 'enzyme';

import * as React from 'react';

import App from './App';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
describe('App SnapShot', () => {

  it('should match snapshot', () => {
      const wrapper = shallow(<App />);
      expect(wrapper.find('div')).toMatchSnapshot();
  });

});
