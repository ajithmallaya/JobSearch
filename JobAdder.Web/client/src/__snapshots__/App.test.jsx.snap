// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App SnapShot should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): ShallowWrapper {
    Symbol(enzyme.__root__): [Circular],
    Symbol(enzyme.__unrendered__): <App />,
    Symbol(enzyme.__renderer__): Object {
      "batchedUpdates": [Function],
      "getNode": [Function],
      "render": [Function],
      "simulateError": [Function],
      "simulateEvent": [Function],
      "unmount": [Function],
    },
    Symbol(enzyme.__node__): Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="App-header"
>
             Job Adder
</div>,
          <div
            className="App-intro"
>
            <Jobs />
</div>,
        ],
        "className": "App",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Job Adder",
            "className": "App-header",
          },
          "ref": null,
          "rendered": " Job Adder",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Jobs />,
            "className": "App-intro",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
    Symbol(enzyme.__nodes__): Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="App-header"
>
               Job Adder
</div>,
            <div
              className="App-intro"
>
              <Jobs />
</div>,
          ],
          "className": "App",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": " Job Adder",
              "className": "App-header",
            },
            "ref": null,
            "rendered": " Job Adder",
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Jobs />,
              "className": "App-intro",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    Symbol(enzyme.__options__): Object {
      "adapter": ReactSixteenAdapter {
        "options": Object {
          "enableComponentDidUpdateOnSetState": true,
          "lifecycles": Object {
            "componentDidUpdate": Object {
              "onSetState": true,
            },
            "getDerivedStateFromProps": true,
            "getSnapshotBeforeUpdate": true,
            "setState": Object {
              "skipsComponentDidUpdateOnNullish": true,
            },
          },
        },
      },
      "attachTo": undefined,
      "hydrateIn": undefined,
    },
  },
  Symbol(enzyme.__unrendered__): null,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="App-header"
>
           Job Adder
</div>,
        <div
          className="App-intro"
>
          <Jobs />
</div>,
      ],
      "className": "App",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " Job Adder",
          "className": "App-header",
        },
        "ref": null,
        "rendered": " Job Adder",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Jobs />,
          "className": "App-intro",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="App-header"
>
             Job Adder
</div>,
          <div
            className="App-intro"
>
            <Jobs />
</div>,
        ],
        "className": "App",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": " Job Adder",
            "className": "App-header",
          },
          "ref": null,
          "rendered": " Job Adder",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Jobs />,
            "className": "App-intro",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " Job Adder",
        "className": "App-header",
      },
      "ref": null,
      "rendered": " Job Adder",
      "type": "div",
    },
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Jobs />,
        "className": "App-intro",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
